testing/trino-product-tests/pom.xml

```diff
@@ -128,6 +128,11 @@
             <artifactId>minio</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-hadoop-toolkit</artifactId>
```

Comment:
By design, Product tests avoid depending on implementation details of the product under test.
Let's remove this dependency. I don't think we need a product test for this at all.

Reply:
Removed product test